### PR TITLE
141/errors trans

### LINKED ISSUE #141: Translate Notification Feedback messages
This might already get advantage of i18n indexing of translations. For example, the keys could be `item.edit.message.failure`, `item.edit.message.success`.

https://www.i18next.com/translation-function/essentials

### PR DESCRIPTION
**Pull request by pyphilia**
close #141

### CODE REVIEW

File: src/components/main/ItemScreen.js [hunk 1]
```diff
@@ -75,7 +77,10 @@ const ItemScreen = () => {
 
   return (
     <Main>
-      <ItemMain item={item}>{content}</ItemMain>
+      <UppyContextProvider enable={enableEdition} itemId={itemId}>
```
Comment (juancarlosfarah) [inline src/components/main/ItemScreen.js:80]: Not specifically for this PR, but in English, this should be `enableEditing`.

File: src/components/file/FileUploader.js [hunk 2]
```diff
@@ -48,53 +40,20 @@ const useStyles = makeStyles((theme) => ({
...
-        itemId,
-        onComplete,
-        onFilesAdded,
-        onError,
-        onUpload,
-      }),
-    );
-  };
+  useEffect(() => {
+    uppy?.on('files-added', () => {
```
Comment (juancarlosfarah) [inline src/components/file/FileUploader.js:53]: Won't `uppy` always have the `on`? Or is this checking if `uppy` is not falsy?

**Review by juancarlosfarah — APPROVED** [review]
Approving! Looks very nice. Much more compact. Just a quick question and an unrelated comment on naming.

File: src/components/file/FileUploader.js (hunk 2 above)
Comment (pyphilia) [reply src/components/file/FileUploader.js:53]: Uppy could be not configured yet.

File: src/components/main/ItemScreen.js (hunk 1 above)
Comment (pyphilia) [reply src/components/main/ItemScreen.js:80]: It's not too much changes, I've applied it 👍

**Review by juancarlosfarah — APPROVED** [review]
👍

**Pull request merged**